pkg/kubectl/daemonset.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+
+	appsv1beta2 "k8s.io/api/apps/v1beta2"
+	"k8s.io/api/core/v1"
+	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
Should not use `extensions/v1beta1` now, since we've bumped to `appsv1beta2`.

Reply:
actually my cluster not support appsv1beta2. i support this appsv1beta2 version, but we should lower version to compatibility？

Reply:
what version is your cluster? This new code is not going to be released before 1.9, and we don't have to support server below 1.8.

Reply:
Yeah, this code should use the new v1 workload api, not extensions.